src/hooks/useStakingAPR.ts

```diff
@@ -0,0 +1,33 @@
+import { useQuery } from '@tanstack/react-query';
+
+import { useEndpointsConfig } from './useEndpointsConfig';
+
+export const useStakingAPR = () => {
+  const { stakingAPR: stakingAPREndpoint } = useEndpointsConfig();
+
+  const queryFn = async () => {
+    if (!stakingAPREndpoint) {
+      return undefined;
+    }
+    const response = await fetch(stakingAPREndpoint, {
+      headers: {
+        'content-type': 'application/json',
+      },
+    });
+
+    const data = await response.json();
+    return data?.[0].rewardRate as number | undefined;
+  };
+
+  const { data } = useQuery({
+    queryKey: ['stakingAPY'],
+    queryFn,
+    enabled: true,
+    refetchOnWindowFocus: false,
+    refetchOnReconnect: false,
+  });
+
+  const formatedAPR = data ? (data * 100).toFixed(2) : undefined;
```

Comment:
nit: can we use `PERCENT_DECIMALS` here? I wonder if it's also technically more correct to be using an `Output` component in the receivers (we could technically make an `AprTag` component that abstracts this away though I guess that adds a bit of work)

Reply:
I tried using output, but the spacing was a bit wierd in the tag, and had an unwanted % suffix, also not sure output is meant to be used in tags 🤔